solr/solr-ref-guide/modules/query-guide/pages/other-parsers.adoc

```diff
@@ -75,6 +75,16 @@ However, unlike `must`, the score of filter queries is ignored.
 Also, these queries are cached in filter cache.
 To avoid caching add either `cache=false` as local parameter, or `"cache":"false"` property to underneath Query DLS Object.
 
+`mm`::
++
+[%autowidth,frame=none]
+|===
+|Optional |Default: `0`
+|===
++
+The number of optional clauses that must match. By default, no optional clauses are necessary for a match
+(unless there are no required clauses). If this parameter is set, then the specified number of `should` clauses is required.
```

Comment:
It's probably worth to clarify that when there are no required clauses it works like mm=1.

Reply:
Thanks for the suggestion @mkhludnev, I updated the description.